appbars/src/main/res/values/styles-topbars.xml

```diff
@@ -57,20 +57,24 @@
     <eat-comment />
 
     <style name="Widget.Vitamin.TopBar.Search" parent="@style/Widget.Vitamin.TopBar">
-        <item name="contentInsetStart">@dimen/vtmn_appbars_topbar_search_content_inset_start</item>
         <item name="materialThemeOverlay">@style/ThemeOverlay.Vitamin.TopBar.Search</item>
     </style>
 
     <style name="ThemeOverlay.Vitamin.TopBar.Search" parent="">
-        <item name="colorOnPrimary">?attr/vtmnContentColorTertiary</item>
+        <item name="colorOnPrimary">?attr/vtmnContentColorPrimary</item>
+        <item name="android:textColorHint">?attr/vtmnContentColorTertiary</item>
     </style>
 
     <style name="Widget.Vitamin.SearchView" parent="@style/Widget.AppCompat.SearchView">
         <item name="closeIcon">@drawable/ic_vtmn_close_line</item>
-        <item name="searchIcon">@drawable/ic_vtmn_search_line</item>
+        <item name="searchIcon">@null</item>
         <item name="searchHintIcon">@drawable/ic_vtmn_search_line</item>
         <item name="goIcon">@drawable/ic_vtmn_check_line</item>
         <item name="voiceIcon">@drawable/ic_vtmn_mic_line</item>
         <item name="iconifiedByDefault">false</item>
+        <item name="android:queryBackground">@android:color/transparent</item>
+        <item name="queryBackground">@android:color/transparent</item>
+        <!-- This negative padding should be removed when a clean way to remove start space will be found -->
+        <item name="android:paddingLeft">-8dp</item>
```

Comment:
Surprised by this line. Vitamin is using the Material padding in the design specification. Does it mean that XML Android styles doesn't follow Material specification? 🤔

Reply:
This case is complicated. The standalone searchView and the topAppBar embedded searchView have really different looks.

When the searchView is embedded in the topAppBar, without specifying anything, the topAppBar adds extra space between the topAppBar left icon and the searchView. 
Additionally to that, the searchView includes extra start and end spacing. 
The result of that is a huge space between the back icon and the searchView (bigger than the space between the back icon and the classic title)

This "hack" must be temporary, a way to reduce this spacing to look more like the spacing between the back icon and the title in a regular topAppBar.
**It should be removed when a solution will be found** (find a way to say the topAppBar to not force spacing when we add a searchView?)